test/controllers/donation_goal_controller_test.exs

```diff
@@ -1,7 +1,7 @@
 defmodule CodeCorps.DonationGoalControllerTest do
   use CodeCorps.ApiCase, resource_name: :donation_goal
 
-  @valid_attrs %{amount: 200, current: false, description: "A description"}
+  @valid_attrs %{amount: 200, description: "A description"}
   @invalid_attrs %{description: nil}
```

Comment:
Nah. `current` is not set by the user, period. It should not be considered invalid, it should just be ignored.